phpstan.neon

```diff
@@ -18,10 +19,48 @@ parameters:
       - *SocialProfileTrait.php
   drupal:
       drupal_root: %currentWorkingDirectory%/core
+      entityMapping:
```

Comment:
We should add inline comments or documentation about this, so that developers know about this is and which will also them adding to this in case they add more entities.

Reply:
There's the tool's documentation which is probably going to be better than anything we can write ourselves: https://github.com/mglaman/phpstan-drupal/#entity-storage-mappings